DSharpPlus.CommandsNext/CommandsNextConfiguration.cs

```diff
@@ -117,6 +117,12 @@ public sealed class CommandsNextConfiguration
         /// </summary>
         public bool IgnoreExtraArguments { internal get; set; } = false;
 
+        /// <summary>
+        /// <para>Gets whether subsequent MessageCreate event handlers are processed after a command was found. If this is set to true, the default command handler marks the event as handled, and subsequent MessageCreate handlers won't be called, otherwise they are called.</para>
```

Comment:
This is wordy and can be simplified. 
```diff
- /// <para>Gets whether subsequent MessageCreate event handlers are processed after a command was found. If this is set to true, the default command handler marks the event as handled, and subsequent MessageCreate handlers won't be called, otherwise they are called.</para>

+ /// <para> Gets or sets whether remaining event handlers will run if a command is successfully parsed.</para>